crates/dbsp/src/tutorial.rs

```diff
@@ -939,7 +939,7 @@
 //! )> {
 //!     let (input_stream, input_handle) =
 //!             circuit.add_input_zset::<Record, i64>();      
-//!     // ... #     let subset = input_stream.filter(|r| {
+//! #     let subset = input_stream.filter(|r| {
```

Comment:
I suspect that the three dots were indicating that there is some more code you can write. But they are not essential.